src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
+      <TodoInfo todo={todo} />
```

Comment:
fix it
https://github.com/mate-academy/react_static-list-of-todos-js/pull/46/files#r1247168369